test/integration/OAuthClientsTest.php

```diff
@@ -120,8 +120,7 @@ function testCreateUserThenGetClient()
         $userCreate = $this->personaClientUser->createUser($gupid, array('name' => 'Sarah Connor', 'email' => $email), $token);
         $user = $this->personaClientUser->getUserByGupid($userCreate['gupids'][0], $token);
         $client = $this->personaClientOAuthClient->getOAuthClient($user['guid'], $token);
-        $this->assertContains('guid', $client);
-        $this->assertContains('scope', $client);
+        $this->assertArrayHasKey('scope', $client);
```

Comment:
`$this->assertContains` is not the correct method to use when checking for a key. This line was throwing a error, but it works correctly if used with `assertArrayHasKey`.